components/layout_2020/flow/construct.rs

```diff
@@ -134,7 +152,7 @@ impl BlockContainer {
                 );
                 return (container, builder.contains_floats);
             }
-            builder.end_ongoing_inline_formatting_context();
+            // builder.end_ongoing_inline_formatting_context();
```

Comment:
Maybe add FIXME comments near those pieces of commented-out code, so we can find them later and distinguish them more easily from "normal" text comments?